dynamic_rest/filters.py

```diff
@@ -513,67 +545,61 @@ def _build_queryset(
         if requirements is None:
             requirements = TreeMap()
 
-        self._get_implicit_requirements(
-            fields,
-            requirements
-        )
+        self._get_implicit_requirements(fields, requirements)
 
         # Implicit requirements (i.e. via `requires`) can potentially
         # include fields that haven't been explicitly included.
         # Such fields would not be in `fields`, so they need to be added.
         implicitly_included = set(requirements.keys()) - set(fields.keys())
         if implicitly_included:
             all_fields = serializer.get_all_fields()
-            fields.update({
-                field: all_fields[field]
-                for field in implicitly_included
-                if field in all_fields
-            })
+            fields.update(
+                {
+                    field: all_fields[field]
+                    for field in implicitly_included
+                    if field in all_fields
+                }
+            )
 
+        nested_filters = None
         if filters is None:
             filters = self._get_requested_filters()
+            nested_filters = (
+                self._get_requested_filters(nested=True)
```

Comment:
#nitpick can we simplify this somehow? calling `_get_requested_filters` to call it again with `nested=True` feels awk

Reply:
Yeah.. it would be ideal if all of the filters were consolidated in one tree as they are before this change. I can see about consolidating them back into one object

Reply:
@sjoanes done, looks less awk now